src/components/StorageOverview/Alerts/Alerts.js

```diff
@@ -1,39 +1,94 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Row, Col } from 'patternfly-react';
 
 import {
   DashboardCard,
   DashboardCardBody,
   DashboardCardHeader,
   DashboardCardTitle,
 } from '../../Dashboard/DashboardCard';
+import { DashboardCardActionsBody } from '../../Dashboard/DashboardCard/DashboardCardActionsBody';
+import { Dropdown } from '../../Form/Dropdown';
+
 import { StorageOverviewContext } from '../StorageOverviewContext';
 import { AlertsBody } from '../../Dashboard/Alert/AlertsBody';
 import { filterAlerts } from './utils';
 import { AlertItem } from '../../Dashboard/Alert/AlertItem';
+import { getAlertSeverity } from '../../../selectors/prometheus/alerts';
+import { ALL, WARNING, CRITICAL } from './strings';
+
+const alertsFilterOptions = [ALL, CRITICAL, WARNING];
 
-export const Alerts = ({ alertsResponse, className }) => {
-  if (!Array.isArray(alertsResponse)) {
-    return null;
+export class Alerts extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      type: alertsFilterOptions[0],
```

Comment:
why not just put `ALL` here?

Reply:
just thought its better to put the reference to the first element instead of hardcoding so that in future if any changes would occur it wont affect that...

Reply:
I see it the other way around. It will be broken if someone changes that array in the future (adding element or changing the order of them). + it will be pretty explicit if we put `ALL` there and we do not need to jump to array to find out about that value.